app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,31 @@
+@mixin forum-post {
+  .user {
+    float: left;
+    margin-right: 1em;
+    text-align: right;
+    width: 7em;
+  }
+
+  .contents {
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.075);
+    line-height: 1;
+
+    // Block formatting context
+    overflow: hidden;
+
+    padding: 20px;
+
+    // Post title
+    .title {
+      margin: -0.4em 0 0.75em 0;
```

Comment:
Properties should be ordered font-size, margin<br>Shorthand form for property `margin` should be written more concisely as `-0.4em 0 0.75em` instead of `-0.4em 0 0.75em 0`